Dockerfile

```diff
@@ -55,6 +56,8 @@ RUN git clone --depth=1 https://github.com/projectcalico/cni-plugin.git $GOPATH/
 WORKDIR $GOPATH/src/github.com/projectcalico/cni-plugin
 RUN git fetch --all --tags --prune
 RUN git checkout tags/${TAG} -b ${TAG}
+COPY dualStack-changes.patch .
+RUN git apply dualStack-changes.patch
```

Comment:
In-place patching regardless of the tag that we're building feels odd to me.

Normally we would fork the upstream repo into the k3s or rancher org, and keep the required patches on a -rancher1 or -k3s1 tag. See https://github.com/rancher/image-build-containerd/blob/master/Makefile#L9-L11